src/types.ts

```diff
@@ -265,24 +271,25 @@ export interface IBadge {
 }
 
 export type Transaction = {
-  type?: string;
-  id: string;
-  createdAt: string;
-  name: string;
-  host: string;
-  title: string | React.ReactNode;
-  subTitle: string | React.ReactNode;
+  amount?: string;
+  boostagram?: Invoice["boostagram"];
+  badges?: IBadge[];
+  createdAt?: string;
+  currency?: string;
   date: string;
-  amount: string;
-  currency: string;
-  value: string;
+  description?: string;
+  host?: string;
+  id: string;
+  location?: string;
+  name?: string;
   preimage: string;
-  badges: IBadge[];
+  subTitle?: string | React.ReactNode;
+  title: string | React.ReactNode;
   totalAmount: string;
-  totalFees: string;
-  description: string;
-  location: string;
-  totalAmountFiat: string;
+  totalAmountFiat?: string;
```

Comment:
those strings are wrong here I think. 
those should be numbers, but I think we currently store the value we get from the connector: in lnd this is a string in lndhub it's a number...

Reply:
Switched it to number

Reply:
string | number would be currently correct.

lnd returns strings (and then we save strings)
lndhub returns numbers (and then we save numbers)

Reply:
Had a look.  
So `totalAmountFiat` on `Transaction` is always a `string.  
`totalAmount` on `Payment` is `number | string`, but we handle this already.
I think we're good here.